src/components/CardPreview.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import {View} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import PropTypes from 'prop-types';
+import styles from '../styles/styles';
+import Text from './Text';
+import usePrivatePersonalDetails from '../hooks/usePrivatePersonalDetails';
+import ONYXKEYS from '../ONYXKEYS';
+import ExpensifyCardImage from '../../assets/images/expensify-card.svg';
+import variables from '../styles/variables';
+
+const propTypes = {
+    /** User's private personal details */
+    privatePersonalDetails: PropTypes.shape({
+        legalFirstName: PropTypes.string,
+        legalLastName: PropTypes.string,
+    }),
+    session: PropTypes.shape({
+        /** Currently logged-in user email */
+        email: PropTypes.string,
+    }),
+};
+
+const defaultProps = {
+    privatePersonalDetails: {
+        legalFirstName: '',
+        legalLastName: '',
+    },
+    session: {
+        email: '',
+    },
+};
+
+function CardPreview({privatePersonalDetails: {legalFirstName, legalLastName}, session: {email}}) {
+    usePrivatePersonalDetails();
```

Comment:
NAB, but maybe this should be `useFetchPrivatePersonalDetails()` or something that maybe better explains what this does. Started some conversation [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1695616206407739) but not a blocker for this PR.